package.json

```diff
@@ -31,10 +35,9 @@
   },
   "devDependencies": {
     "babel-standalone": "~6.17.0",
-    "carbon-factory": "3.3.3",
+    "carbon-factory": "git://github.com/Sage/carbon-factory.git#build-webpack",
```

Comment:
Can this be updated yet to point to a non-branch version?

Reply:
ah yes it can